bpfman/src/bin/cli/args.rs

```diff
@@ -36,6 +36,10 @@ pub(crate) enum Commands {
     /// eBPF Bytecode Image related commands.
     #[command(subcommand)]
     Image(ImageSubCommand),
+    #[clap(hide = true)]
+    Man(crate::manpage::Args),
+    #[clap(hide = true)]
```

Comment:
I may come back later and add helptext to both commands. Even though it is hidden, manpages are still generated and `sudo bpfman man --help` still outputs data.

Reply:
I did it quickly. I also skipped generating man pages for hidden commands.
Completions will only work for hidden commands if you start typing them, otherwise they aren't suggested.